docs/gigabyte-rgb-fusion2-guide.md

```diff
@@ -29,8 +29,8 @@ Gigabyte RGB Fusion 2.0 5702 Controller
 
 ## Lighting
 
-The controllers support six color modes: `off`, `fixed`, `pulse`, `flash`,
-`double-flash` and `color-cycle`.
+The controllers have built-in support for six color modes: `off`, `fixed`, `pulse`, `flash`, `double-flash` and `color-cycle`. The `color-cycle` mode fades between all
+color hues, but the extra `fusion_rgb_cycle` script can be used to fade between specific colors.
```

Comment:
For a new user, this now contradicts a later sentence:

> The more elaborate color/animation schemes supported by the motherboard on the
addressable headers are not currently supported.

Also, I think the complete path should be used, with a link:

```diff
- `fusion_rgb_cycle`
+ [`extra/contrib/fusion_rgb_cycle.py`]
   [ ... ]
+ [`extra/contrib/fusion_rgb_cycle.py`]: ../extra/contrib/fusion_rgb_cycle.py
```



---

(Nitpick:) in the same paragraph, either hard wrap at ~80 columns (strongly preferred), or don't hard wrap at all.

Reply:
My reason for putting it where I did was to give the user a summary of their options for patterns in one place. I think that sentence is still broadly true (for instance the motherboard supports an audio-driven color scheme I think), but I've tweaked it to be clear there :)

Reply:
(Take a look at the current diff and see if that seems good?)

Reply:
That makes sense.  When I first read it really didn't raise any flags, it's just later that I noticed what a new user could find a bit confusing.  But it's good now, thanks for the change.